app-backend/tile/src/it/resources/application.conf

```diff
@@ -0,0 +1,18 @@
+gatling {
+  host = "https://app.staging.rasterfoundry.com"
+
+  refreshToken = ""
+  refreshToken = ${?REFRESH_TOKEN}
+
+  projectId = ""
+  projectId = ${?PROJECT_ID}
+
+  randomSeed = 42
+
+  tms {
+    template = "/tiles/${projectId}/$z/$x/$y?token=${authToken}"
```

Comment:
So I think we should actually be able to use `mapToken` here instead and that will get rid of the need to do anything with refresh tokens